cutecom-ng.pro

```diff
@@ -19,7 +19,11 @@ SOURCES += main.cpp\
     outputmanager.cpp \
     historycombobox.cpp \
     history.cpp \
-    searchhighlighter.cpp
+    searchhighlighter.cpp \
+    xmodem.cpp \
+    crc16.cpp \
```

Comment:
potrebbe aver senso mettere le librerie di terze parti in un'altra directory, per esempio `libs`.